src/services/genre_detector.py

```
@@ -2,36 +2,53 @@
-            
-            text = ' '.join([
-                entry.get('title', '') + ' ' + entry.get('description', '')
-                for entry in feed.entries[:5]
-            ]).lower()
-            
+
+            text = " ".join(
+                [
+                    entry.get("title", "") + " " + entry.get("description", "")
+                    for entry in feed.entries[:5]
+                ]
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)